google-cloud-bigtable/pom.xml

```diff
@@ -25,7 +25,7 @@
     <skipTests>false</skipTests>
     <skipUnitTests>${skipTests}</skipUnitTests>
     <skipITs>${skipTests}</skipITs>
-
+    <java-log-config>src/test/resources/logging.properties</java-log-config>
```

Comment:
Please dont group 1/2 the logging related changes with test skipping and other 1/2 with dep versions.

Instead create a new section under test skipping with both of the logging properties with comments explaining what the properties do:

```
<!-- Enable the ability to skip ....
<skipTests>false</skipTests>
....

<!-- Configure test logging output. By default tests print info logs to stdout/err.
However for kokoro runs to enable better debugging of connection failures, the `enable-verbose-grpc-logs` profile is enabled. It will enable verbose logs and redirect output to a test specific file in the integration tests report directory.  These propeties should only be set by the `enable-verbose-grpc-logs`  profile -->
<bigtable.enable-grpc-logs>false</bigtable.enable-grpc-logs>
<java-log-config>src/test/resources/logging.properties</java-log-config>
```